src/redux/actions/sessionActions.js

```diff
@@ -20,7 +20,6 @@ import {
 // logs in the user via a firebase token
 // fetches the user data upon success and redirects to the app
 export const sessionUserLogin = (credentials, history) => (dispatch) => {
-  dispatch({type: SESSION_LOADING_LANDING, payload: true});
```

Comment:
Why get rid of this? Curious

Reply:
Automatic loading regardless of condition